docs/preview/yamlgettingstarted-triggers.md

```diff
@@ -50,3 +50,82 @@ Note, path filters are only supported for Git repositories in VSTS.
 Continuous integration builds can be turned off by specifying `trigger: none`
 
 Optionally, the triggers can be managed from the web definition editor, on the Triggers tab.
+
+
+## Trigger syntax for resource types  (for discussion only)
+
+CD triggers are configured per resource. 
+
+For repository resource type, syntax for trigger: 
+
+```yaml
+resources:
+  repositories:
+  - repository: string
+    type: git
+    trigger:
+      branches:
+        include: [string]
+        exclude: [string]
+      paths:
+        include: [string]
+        exclude: [string]
+...
+
+For example:
+
+```yaml
+resources:
+  repositories:
+  - repository: mygitrepo
+    type: git
+    trigger:
+      branches:
+        include:
+        - master
+        - releases/*
+        exclude:
+        - releases/old*
+```
+
+
+For build resource type, syntax for trigger
+
+```yaml
+resources:
+  builds:
+  - build: string
+    type: build  
+    trigger:  
+      include:
+      - branch: string
+        tags: [string]  # build tags
+      exclude:      
+      - branch: string
+        tags: [string]
+```
+
+For example,
+
+```yaml
+resources:
+  builds:
+  - build: myBuild
+    type: build  
+    trigger:  
+      include:
+      - branch: master
```

Comment:
I'm not intimately familiar with the nuances of git. Could someone specify... 

```yaml
Include:
  - branch: refs/pull/*
```

... And then enable resource authorization for their own pr?